35/generated/mirror_checkpoint_config.html

```diff
@@ -410,6 +410,16 @@ <h4><a id="replication.policy.class"></a><a id="mirror_checkpoint_replication.po
 </tbody></table>
 </li>
 <li>
+<h4><a id="replication.policy.internal.topic.separator.enabled"></a><a id="mirror_checkpoint_replication.policy.internal.topic.separator.enabled" href="#mirror_checkpoint_replication.policy.internal.topic.separator.enabled">replication.policy.internal.topic.separator.enabled</a></h4>
+<p>Whether to use replication.policy.separator to control the names of topics used for checkpoints and offset syncs. By default, custom separators are used in these topic names; however, if upgrading MirrorMaker 2 from older versions that did not allow for these topic names to be customized, it may be necessary to set this property to 'false' in order to continue using the same names for those topics.</p>
+<table><tbody>
+<tr><th>Type:</th><td>boolean</td></tr>
+<tr><th>Default:</th><td>true</td></tr>
+<tr><th>Valid Values:</th><td></td></tr>
+<tr><th>Importance:</th><td>low</td></tr>
+</tbody></table>
+</li>
+<li>
```

Comment:
Was this a forgotten documentation chunk that we forgot to add until now?

Reply:
No, https://issues.apache.org/jira/browse/KAFKA-15102 fixes for 3.5.2 and many other versions.